setup.pp

```diff
@@ -110,7 +110,7 @@
   user    => 'pi',
   path    => '/bin:/usr/bin',
   command => 'python3 /home/pi/PiWeatherRock/scripts/upgrade.py',
-  unless  => 'grep "0.0.14" config.json',
+  unless  => 'grep $(cat version.py|cut -d "\'" -f2) config.json',
```

Comment:
I'm getting the following error when I test line 113 as a standalone bash script:
```
cut: the delimiter must be a single character
```
I've had success with this, though: `grep $(cat version.py|cut -d"'" -f2) config.json`